nvchecker/run-nvchecker.cron

```diff
@@ -9,5 +9,6 @@ cd "$CHECKOUT"
 git stash
 git pull -q
 git stash pop
+nvchecker/gen-config.sh
```

Comment:
We should catch errors here explicitly or make the whole script run with the `-e` option.

Reply:
8074a9ce14cfaf79618a4f9ddc1e5c633e7ab407